servers/audio/audio_effect.h

```diff
@@ -47,6 +47,7 @@ class AudioEffectInstance : public RefCounted {
 public:
 	virtual void process(const AudioFrame *p_src_frames, AudioFrame *p_dst_frames, int p_frame_count);
 	virtual bool process_silence() const;
+	virtual void set_current_channel(int p_channel) {};
```

Comment:
```suggestion
	virtual void set_current_channel(int p_channel) {}
```
For format check

Reply:
Fixed. Also added `virtual` to compressor and capture `set_current_channel` (I received email suggestions for these, but don't see them on github). I didn't add it to record as its `set_current_channel` is a bit more nuanced, and if someone wants to override it, they can refactor it at that time.

Reply:
We always add both, not adding it won't stop overriding later you need `final` for that